.circleci/config.yml

```diff
@@ -57,6 +57,14 @@ jobs:
             npm install -g publish
             publish
 
+      - run:
+        - name: Publish to npm public registry
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
you need to migrate this repo to start using GitHub actions first. @lchan131 should know more about this